README.md

```diff
@@ -177,9 +177,9 @@ client.create_service service`
 ```
 
 #### Update an entity
-For example: `update_pod`, `update_service`, `update_replication_controller`, `update_secret`
+For example: `update_pod`, `update_service`, `update_replication_controller`, `update_secret`, `update_resource_quota`, `update_limit_range`
 
-Input parameter - object of type `Service`, `Pod`, `ReplicationController`
+Input parameter - object of type `Pod`, `Service`, `ReplicationController` etc.
```

Comment:
> to align with the text above: update_pod, update_service.
> do you want this removed?

in the future please separate changes like this to a separate patch since it is unrelated and it confuses when reviewing.
but nonetheless such alignment fixes are welcome.